docs/browser-compat-data.js

```diff
@@ -0,0 +1,476 @@
+    const latestVersion = this.getLatestVersion();
+    
+    document.getElementById('total-apis').textContent = totalAPIs;
+    document.getElementById('stable-apis').textContent = stableAPIs;
+    document.getElementById('experimental-apis').textContent = experimentalAPIs;
+    document.getElementById('latest-version').textContent = latestVersion;
+  }
+
+  getLatestVersion() {
+    const versions = this.compatData
+      .map(api => api.version)
+      .filter(v => v !== 'Unknown')
+      .sort((a, b) => this.compareVersions(b, a));
+    
+    return versions[0] || this.currentVersion || 'Unknown';
+  }
+
+  renderAPIs() {
+    const container = document.getElementById('api-container');
+    const emptyState = document.getElementById('empty-state');
+    
+    if (!this.filteredData || this.filteredData.length === 0) {
+      container.innerHTML = '';
+      if (emptyState) emptyState.classList.remove('hidden');
+      return;
+    }
+    
+    if (emptyState) emptyState.classList.add('hidden');
+    
+    // Group APIs by parent/category for better organization
+    const grouped = this.groupAPIsByParent(this.filteredData);
+    
+    container.innerHTML = Object.keys(grouped)
+      .sort() // Sort groups alphabetically
+      .map(groupName => {
+        const apis = grouped[groupName];
+        return this.renderAPIGroup(groupName, apis);
+      }).join('');
+  }
+
+  groupAPIsByParent(apis) {
+    const grouped = {};
+    
+    apis.forEach(api => {
+      const key = api.parentName || api.category;
+      if (!grouped[key]) {
+        grouped[key] = [];
+      }
+      grouped[key].push(api);
+    });
+    
+    return grouped;
+  }
+
+  renderAPIGroup(groupName, apis) {
+    const categoryColor = this.getCategoryColor(apis[0].category);
+    const categoryIcon = this.getCategoryIcon(apis[0].category);
+    
+    return `
+      <div class="api-group card rounded-lg shadow-sm ${categoryColor} overflow-hidden" style="border-left: 4px solid;">
+        <div class="p-6">
+          <div class="flex items-center justify-between mb-4">
+            <h3 class="text-lg font-semibold text-primary flex items-center gap-2">
+              <span>${categoryIcon}</span>
+              ${groupName}
+            </h3>
+            <span class="badge bg-secondary text-secondary">
+              ${apis.length} API${apis.length > 1 ? 's' : ''}
+            </span>
+          </div>
+          
+          <div class="api-list space-y-2">
+            ${apis.map(api => this.renderAPIItem(api)).join('')}
+          </div>
+        </div>
+      </div>
+    `;
+  }
+
+  renderAPIItem(api) {
+    const statusBadge = this.getStatusBadge(api);
+    const versionBadge = `<span class="version-badge">${api.version}</span>`;
+    
+    return `
+      <div class="api-item-row">
+        <div class="api-main-info">
+          <div class="api-name-section">
+            <h4 class="api-name">${api.name}</h4>
```

Comment:
You need to escape this if needed, because HTML elements' name seems to be: `<audio>`